### PR TITLE
Add profile for running Hive S3 tests

### PR DESCRIPTION
**Pull request by sopel39**

### CODE REVIEW

File: presto-hive-hadoop2/pom.xml [hunk 1]
```diff
@@ -130,5 +130,25 @@
...
+
+        <profile>
+            <id>test-hive-hadoop2-s3</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-surefire-plugin</artifactId>
+                        <configuration>
+                            <excludedGroups>hive</excludedGroups>
```
Comment (findepi) [inline presto-hive-hadoop2/pom.xml]: excluded?

Comment (electrum) [reply presto-hive-hadoop2/pom.xml]: If we don't exclude here, this profile will run all the Hive tests. But if we do exclude, you can't specify both profiles to run all tests (they will counteract each other) -- you'd need a third profile.

For using a CI, having it run both (no exclude) is probably better. For just testing S3, exclude is needed. I'm fine either way. (I run them from IntelliJ)

Comment (electrum) [reply presto-hive-hadoop2/pom.xml]: Is it possible to use Maven property here? If so, no need for a separate profile, just override the excluded groups list when running Maven.

Comment (sopel39) [reply presto-hive-hadoop2/pom.xml]: I changed the approach a bit. Not there is a default profile that excludes both `hive` and `hive-s3` group. The other profiles either exclude `hive` or `hive-s3` group. This way you can specify both profiles and both `hive` and `hive-s3` would execute.

**Comment by sopel39** [comment]
@electrum @findepi could you take another look?

File: presto-hive-hadoop2/pom.xml [hunk 2]
```diff
@@ -98,19 +98,24 @@
...
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-surefire-plugin</artifactId>
-                <configuration>
-                    <excludedGroups>hive,hive-s3</excludedGroups>
-                </configuration>
-            </plugin>
-        </plugins>
-    </build>
-
     <profiles>
```
Comment (kokosing) [inline presto-hive-hadoop2/pom.xml]: You could create a README.md file in this module which would explain how to run this

File: presto-hive-hadoop2/pom.xml [hunk 3]
```diff
@@ -119,7 +124,26 @@
                         <groupId>org.apache.maven.plugins</groupId>
                         <artifactId>maven-surefire-plugin</artifactId>
                         <configuration>
-                            <excludedGroups>hive-s3</excludedGroups>
+                            <groups>hive</groups>
```
Comment (kokosing) [inline presto-hive-hadoop2/pom.xml]: previously it was running all but s3 tests, now it is running only `hive` tests.

To me it is ok. I do not know what other think about it.

File: presto-hive-hadoop2/pom.xml (hunk 2 above)
Comment (findepi) [reply presto-hive-hadoop2/pom.xml]: unlikely to be found ;)

File: presto-hive-hadoop2/pom.xml (hunk 3 above)
Comment (sopel39) [reply presto-hive-hadoop2/pom.xml]: There are just two test classes here, `hive` and `hive-s3`

**Pull request merged**